cilium/cmd/monitor.go

```diff
@@ -102,9 +102,7 @@ func openMonitorSock(path string) (conn net.Conn, version listener.Version, err
 		conn, err = net.Dial("unix", path)
 		if err == nil {
 			version = listener.Version1_2
-			if strings.HasSuffix(path, "monitor.sock") {
-				version = listener.Version1_0
-			}
+	                // v1.0 socket deprecated and removed
```

Comment:
I don't think that this comment is necessary, it doesn't simplify or help to explain functionality that future developers may need to reference. Similarly for the other places in this PR that mention the v1.0, please just remove them.

Reply:
Sure will remove it.

Reply:
Addressed review comments.